velox/exec/RowNumber.cpp

```diff
@@ -379,8 +351,15 @@ void RowNumber::reclaim(
     return;
   }
 
-  if (inputSpiller_ != nullptr) {
-    // Already spilled.
+  // TODO: Add a spill level limit for RowNumber spill.
+  if (spillConfig_->exceedJoinSpillLevelLimit(spillPartitionBits_.begin())) {
```

Comment:
> exceedJoinSpillLevelLimit

Are we re-using join's limit? Let's add a limit for rownumber or rename join's config so it can be used in any operator.

Reply:
I've created a PR to config it, could you help to take a look? https://github.com/facebookincubator/velox/pull/8978